nethserver-collabora.spec

```diff
@@ -6,7 +6,7 @@ License: GPL
 Source: %{name}-%{version}.tar.gz
 BuildArch: noarch
 
-Requires: coolwsd CODE-brand
+Requires: coolwsd CODE-brand >= 21.11
```

Comment:
Could you please make just one line to require CODE it will be more clear

Requires: coolwsd 
Requires: CODE-brand >= 21.11